synfig-studio/src/gui/app.cpp

```diff
@@ -1057,13 +1057,32 @@ DEFINE_ACTION("help-about",     Gtk::StockID("synfig-about"))
 // actions: Keyframe
 DEFINE_ACTION("keyframe-properties", _("Properties"))
 
+// actions: move to tab
+DEFINE_ACTION("move-to-tab-1",  _("Move to Tab 1"))
```

Comment:
1. The title does not match the content of the PR.
2. I don't know why using a loop to generate names in one place you didn't use it in another.

Reply:
Every `_()` call adds a string to translation. Less strings to translate - less work need to be done.

Reply:
oh ok now it makes much more sense and the numbers dont get translated anyways, perfect!